resources/examples/manager/BasicAssetLibrary/plugin/BasicAssetLibrary/BasicAssetLibraryInterface.py

```diff
@@ -110,11 +110,18 @@ def entityExists(self, entityRefs, context, hostSession):
             results.append(result)
         return results
 
-    def resolve(self, entityRefs, traitSet, context, hostSession):
+    def resolve(self, entityRefs, traitSet, context, hostSession, successCb, errorCb):
```

Comment:
[consider] Surprised `pylint` didn't moan about renaming the parameters here, worth matching the base class names?

Reply:
Turns out we don't run pylint on the examples. Trying to pylint the `resources` directory gives lots of issues, mostly about the lack of camelCase conforming, but lots of other errors too.

Reply:
Fixed in a9ed554b then realised I'd forgot an argument and fixed in 0c8e7f08

Reply:
Oh good spot! Sorry that other one has been there for ages 🙃

Reply:
I forgot about not linting too - it was as we wanted to use pep8 - and local pylint was harder!